library/src/G4CMPPhononBoundaryProcess.cc

```diff
@@ -410,3 +419,50 @@ GetLambertianVector(const G4ThreeVector& surfNorm, G4int mode) const {
 
   return reflectedKDir;
 }
+
+
+// Get the position on the edge of two surfaces
+
+G4ThreeVector G4CMPPhononBoundaryProcess::
```

Comment:
See my comment in the .hh file about passing in non-const reference and modifying in place.